tests/assets/rrweb-instrumented.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2020 New Relic Corporation.
+  PDX-License-Identifier: Apache-2.0
+-->
+<html>
+<head>
+  <title>RUM Unit Test</title>
+  <style>
+    .left{
+      position: absolute; left: 50px; top: 200px;
+    }
+    .right {
+      position: absolute; right: 50px; top: 200px;
+    }
+  </style>
+  {init}
+  {config}
+  {loader}
+  {script-injection}
+</head>
+<body>
+  this is a generic page that is instrumented by the JS agent
```

Comment:
Might change this to indicate this page has elements specific to rrweb implementation.